backend/src/core/models/games.aggregate.js

```diff
@@ -8,9 +8,7 @@ export class GamesAggregate {
   }
 
   get content() {
-    const contentCopy = new GamesAggregate(this.#games).#games;
-
-    return contentCopy;
+    return new GamesAggregate(structuredClone(this.#games)).#games;
```

Comment:
Wow cool thing man! You found some hot new thing hehe awesome! No we don't need lodash anymore :) :1st_place_medal: